compiler/basicTypes/Multiplicity.hs

```diff
@@ -27,19 +25,24 @@ module Multiplicity
   , submult
   , traverseMult
   , multThingList
-  , mapMult ) where
+  , mapMult
+  , fromMult
+  , toMult ) where
 
 import GhcPrelude
 
 import Data.Data
 import Outputable
+import {-# SOURCE #-} TyCoRep (Type)
+import {-# SOURCE #-} TysWiredIn ( oneDataConTy, omegaDataConTy )
+import {-# SOURCE #-} Type( eqType )
```

Comment:
:slightly_frowning_face:

I must confess that I was rather happy to have removed `Multiplicity` from the `Type` recursive block. But well… I guess it can't be helped.